docs/components/typography/test/headings.html

```diff
@@ -0,0 +1,57 @@
+---
+title: Typography
+minver: 0.1.6
+---
+{% extends 'component.njk' %}
+{% block content %}
+  <section><!-- Headings -->
+    <h2 id="headers">Headings</h2>
+    <div class="demo" data-visreg="typography-headings">
```

Comment:
Some notes:

1. Drop visreg for elements that aren't going to render useful results for snapshot output for now.
2. Make a story to split up `data-visreg` into `data-visual-diff` and `data-dom-diff` so we can get just snapshot tests

Reply:
https://jira.rax.io/browse/SURF-1086